source/parsed_kdtree_distance.cc

```diff
@@ -51,8 +51,11 @@ double ParsedKDTreeDistance<dim>::value(const Point<dim> &p, const unsigned int
   kdtree->findNeighbors(resultSet, &p[0], nanoflann::SearchParams());
   //index.knnSearch(query, indices, dists, num_results, mrpt_flann::SearchParams(10));
 
+  // silence a warning about unused variable
+  // when compiling in release mode
+  (void)component;
```

Comment:
I write here since I cannot comment on above lines. The first assert of the function value is

```
   // Only 1 component in this function
   AssertDimension(component, 0);
```
and `component` is a `const unsigned int`. 
according to the assert, the only allowed value for `component` is zero. Is this right? If so, why we need an argument which can takes only one value and is only used in an assert?